docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-menu.mdx

```diff
@@ -137,18 +140,19 @@ fun SelectedMessageMenu(
     headerContent: @Composable ColumnScope.() -> Unit = {
         // Header content
     },
-    bodyContent: @Composable ColumnScope.() -> Unit = {
-        // Body content
+    centerContent: @Composable ColumnScope.() -> Unit = {
+        // Center content
     }
 )
 ```
-* `headerContent`: Allows you to customize what content you show on the top part of the menu. By default it shows `ReactionOptions`
-* `bodyContent`: Allows you to customize what content you show on the bottom part of the menu. By default it shows `MessageOptions`
+
+* `headerContent`: Allows you to customize the content shown at the top of the menu. By default it shows `ReactionOptions`.
```

Comment:
I think our newer unofficial guidelines are to avoid mentioning exact implementations like these since they are inside of `DefaultX`. 

Using "reactions options" instead of `ReactionOptions` might be advisable.

Please double check this with @filbabic

Reply:
Yeah this is what we should do - have it as reaction otpions (separated, lowercase) so it doesn't show the specific component, in case we rename it., same for all other items!